PREINSTALL.md

```diff
@@ -0,0 +1,42 @@
+Use this extension to synchronize documents from a Cloud Firestore collection to a Meilisearch index. This allows you to use a full text search in your Cloud Firestore documents.
+
+This extension listens to each creation, update or deletion of your documents to keep them sync with your Meilisearch index, so the data in Meilisearch is a mirror of your content in Cloud Firestore. You can then run queries on this mirrored dataset.
+
+Note that this extension only listens for changes to _documents_ in a specific collection, but not changes in any _subcollection_. However, you can install additional instances of this extension to listen to other collections in your Firestore database.
```

Comment:
> but not changes in any _subcollection_ 

Not sure I understand.

Reply:
You had the possibility to create [subcollections in Firestore](https://firebase.google.com/docs/firestore/data-model#subcollections). For example, you create a collections `movies` you can create fields for each document (movie) but you can also create a `Collection` inside the `document`, It is to facilitate the nesting of collection and reference.
So if you create a subcollection inside a document in Firestore it will not be exported to `Meilisearch` instead you can install a second extension for this specific `subcollection`.